Source/CsDebugScript.Engine/Engine/IDebuggerEngine.cs

```diff
@@ -288,5 +288,38 @@ public interface IDebuggerEngine
         /// </summary>
         /// <returns></returns>
         DebugEventInfo GetLastEventInfo();
+
+        /// <summary>
+        /// When doing live process debugging continues debugee execution of the current process.
+        /// </summary>
+        void ContinueExecution();
```

Comment:
This interface should be for implementation only. You shouldn't add all variations of overloaded functions in the interface itself (as all classes that implement it should do that). Instead, please use Debugger static class to add all overloaded methods (including one with all arguments for convenience and possibility to make this interface internal one day)...